include/mbedtls/gcm.h

```diff
@@ -147,16 +160,18 @@ int mbedtls_gcm_auth_decrypt( mbedtls_gcm_context *ctx,
                       unsigned char *output );
 
 /**
- * \brief           Generic GCM stream start function
- *
- * \param ctx       GCM context
- * \param mode      MBEDTLS_GCM_ENCRYPT or MBEDTLS_GCM_DECRYPT
- * \param iv        initialization vector
- * \param iv_len    length of IV
- * \param add       additional data (or NULL if length is 0)
- * \param add_len   length of additional data
- *
- * \return         0 if successful
+ * \brief           This function sets the GCM key, and prepares to authenticate 
```

Comment:
Blocker: This function doesn't set the key.
Suggestion: As for other incremental operations such as message-digests, use a wording like `This function starts a GCM encryption or decryption operation.`

Reply:
fixed